lib/model/bloc/home/HomeViewModelProvider.dart

```diff
@@ -23,7 +28,14 @@ class HomeViewModelProvider implements ViewModelProvider<HomeViewModel> {
   HomeViewModel _snapshot;
   ProfileEntity _lastProfile;
   ProfileRepositoryInterface _profileRepository;
+  bool _isDebugBuild;
+
   HomeViewModelProvider(this._accountRepository);
+
+  void init(BuildContext context) {
```

Comment:
not sure about this, now the view model provider. needs the build context

Reply:
push out the app config dependency. maybe just have a constructor for this provider that has the debug flag passed in on construction.

Reply:
Sorted out